### DISCUSSION TITLE
Providing multiple profiles to Application Service i.e rules-engine and export-mqtt

### DISCUSSION THREAD

**Discussion by Devendra-7** [edgexfoundry/community#276] [Q&A] [answered]
Hi Team,
I have to provide two profiles to application service, while starting the service in non secure mode i.e rule-engine and export-mqtt. Rules engine for filtering the data, after data filtering need to export the same data to the cloud like AWS/ Azure using mqtt.

Can i start the application service two times by proving the rule-engine profile first and secondly export-mqtt profile, one after another or crate a single profile with rule-engine and export-mqtt support, then start the application service with single configuration file.

Please suggest the right way for providing the different profiles for a single service.

Thanks in Advance !

**Comment by cloudxxx8** [comment, marked as answer]
No, you should start two different app services for the two purpose.
You can easily define them in the docker-compose file.
